RELEASE_NOTES.md

```diff
@@ -2,23 +2,81 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all of the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 2.0.0
+
+This is a major release of Refinery with a new configuration file format, new samplers, and updated runtime behavior. It has several breaking changes, and so it also comes with a conversion tool.
+
+### Configuration File Changes
+
+The configuration and sampler file formats hav been completely redesigned. Key changes:
+
+- The preferred file format is now YAML instead of TOML. All examples and documentation now use YAML format (the old *_complete.toml files now have 1.x in their names).
+- The config file is now organized into sections for clarity.
+- Sampler rules now require default target named `__default__` to specify the default sampling behavior.
+- Many default values have changed to be more useful.
+- Configurations are now fully validated - misspellings, type errors, faulty indentions, and extra values are now detected. Refinery will no longer run if your configurations are invalid.
+- Documentation for configuration is now automatically generated so that it will stay in sync with the source.
+- A conversion tool has been provided to convert a v1.x configuration file to the new format.
```

Comment:
@TylerHelmuth I would actually move the configuration conversion tool bullet point here to being the 2nd bullet point in the list. As a user, reading "You have to use YAML now instead of TOML" would discourage me until I realize there's a conversion tool available.

Reply:
Great idea.

Reply:
Done.